cli-commands/forgae-shape/shape-vue-cmd.js

```diff
@@ -1,24 +1,50 @@
+const util = require('util');
```

Comment:
change the folder name to "forgae-shapes" because this will not be the only shape